internal/cache/file/cache_handle_test.go

```diff
@@ -484,7 +590,7 @@ func (cht *cacheHandleTest) Test_Read_RandomWithNoRandomDownload() {
 	n, cacheHit, err := cht.cacheHandle.Read(context.Background(), cht.bucket, cht.object, offset, dst)
 
 	jobStatus := cht.cacheHandle.fileDownloadJob.GetStatus()
-	ExpectEq(jobStatus.Name, downloader.NOT_STARTED)
+	ExpectEq(jobStatus.Name, downloader.NotStarted)
```

Comment:
nit: we need to change the order of input parameter, as we did below in line: 606.